domains/namesmt.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NamesMT",
+        "email": "dangquoctrung123@gmail.com"
+    },
+    "record": {
+        "CNAME": "namesmt.github.io"
```

Comment:
site gives 404 error (if you do have the website hosted then send website link in this PR)

Reply:
Hi, the main page is expected to be 404 for now, but I will use it to host sub-projects like https://namesmt.github.io/vue-modern-cropper/